sdk/java-sdk/src/main/java/kalix/javasdk/eventsourcedentity/EventSourcedEntity.java

```diff
@@ -129,6 +129,8 @@ protected final Effect.Builder<S> effects() {
 
       OnSuccessBuilder<S> emitEvents(List<?> event);
 
+      OnSuccessBuilder<S> deleteEntity(Object finalEvent);
```

Comment:
if I'm not mistaken this is also used for the Spring SDK and nothing special is needed for Spring SDK?